src/jit/lsraarm.cpp

```diff
@@ -717,6 +717,11 @@ void Lowering::TreeNodeInfoInit(GenTree* tree)
             info->internalIntCount = 1;
             break;
 
+        case GT_PUTARG_SPLIT:
+            info->srcCount = 0;
+            info->dstCount = 0;
```

Comment:
`GT_PUTARG_SPLIT` does not generate any code. So there's not need srcCount and dstCount. Registers are produced by `GT_PUTARG_REG`: the operand of `FIELD_LIST` and this `FIELD_LIST` is operand of `GT_PUTARG_SPLIT`.